packages/data-context/src/data/ProjectLifecycleManager.ts

```diff
@@ -291,6 +291,20 @@ export class ProjectLifecycleManager {
    *  3. The first browser found.
    */
   async setInitialActiveBrowser () {
+    const browsers = await this.ctx.browser.allBrowsers()
+    const configBrowser = String(this.loadedFullConfig?.browser)
```

Comment:
I don't know if we need `String()` here. Can we find a way to avoid checking against `'undefined'`?

Reply:
Just fixed this